### PR TITLE
Update rig utilities documentation

### PR DESCRIPTION
**Pull request by mossblaser**
* Fixed the page title
* Added a couple of introductory sentences
* s/monotonic/atomic/ (whoops...)

### CODE REVIEW

File: docs/source/utility_apps.rst [hunk 1]
```diff
@@ -1,5 +1,12 @@
+Standalone Utility Applications
+===============================
+
+The following command-line utility applications are installed alongside the Rig
+Python library. These utilities are simple wrappers around the Rig API with an
+emphasis on common system and debug tasks.
+
 ``rig-boot``
-============
+------------
```
Comment (mundya) [inline docs/source/utility_apps.rst]: Weirdly, I had *just* noticed this mistake.

**Comment by mundya** [comment]
LGTM

Rigged

**Comment by mossblaser** [comment]
The link to `rig-counters` in your [other PR](https://github.com/project-rig/nengo_spinnaker/pull/44#issuecomment-132931039) caught my eye and I thought I may as well get it fixed now rather than forgetting about it.

**Comment by mundya** [comment]
I did wonder.  Anyway, feel free to merge when Travis ensures that you haven't trampled on my super-secret project embedded in the documentation.

**Pull request merged**